src/xdocs/checks/coding/fallthrough.xml

```diff
@@ -161,33 +120,74 @@ switch (i) {
             Example:
         </p>
         <source>
-switch (i) {
-  case 1:
-    break;
-  case 2: // Previous case: OK
-          // This case: violation, last case must have statement that transfer control
-    i++;
+class Example2 {
+  public int foo() {
+    int i = 0;
+    while(i &gt;= 0) {
+      switch(i) {
+        case 1:
+          i++;  // falls-thru
+        case 2: // violation
+          i++;
+          break;
+        case 3:
+          i++;
+          return;
+        case 4:
+          i++;
+          throw new Exception();
+        case 5:
+          i++;
+          continue;
+        case 6:
+        case 7:
+        case 8:
+          i++;
+        case 11:
+          i++; // fall through
+      }
+    }
+  }
 }
         </source>
         <p>
             To configure the check with custom relief pattern:
         </p>
         <source>
 &lt;module name="FallThrough"&gt;
-   &lt;property name="reliefPattern" value="FALL?[ -]?THROUGH"/&gt;
+   &lt;property name="reliefPattern" value="no break by design"/&gt;
 &lt;/module&gt;
         </source>
         <p>Example:</p>
         <source>
-switch (i) {
-  case 1:
-    i++;
-    // FALL-THROUGH
-  case 2: // OK, "FALL-THROUGH" matches the regular expression "FALL?[ -]?THROUGH"
-    i++;
-    // fall-through
-  case 3: // violation, "fall-through" doesn't match
-    break;
+class Example3 {
+  public int foo() {
+    int i = 0;
+    while(i &gt;= 0) {
+      switch(i) {
+        case 1:
+            i++; // FALL THROUGH
```

Comment:
Make a comment as `no break by design` to show that violation disappear for special config

Reply:
Done